README.md

```diff
@@ -49,7 +49,7 @@ The file ReadMe.txt in a build bundle describes how to install the library on th
    16. [Cross Platform Build and Test on Linux for MinGW](#2-16)
    17. [Building with C++11 and other compilers](#2-17)
    18. [Static and Shared Libraries](#2-18)
-   19. [Support for bmff files (CR3, HEIF, HEIC, and AVIF)](#2-19)
+   19. [Support for bmff files (e.g., AVIF, CR3, HEIF, HEIC, and JXL)](#2-19)
```

Comment:
No comma after "e.g."?

Reply:
> No comma after "e.g."?

I didn't know about the use of [serial commas](https://en.wikipedia.org/wiki/Serial_comma) in US English. In British English, apart from ambiguities, they are usually removed. I will use serial commas in exiv2 docs in the future.